### ISSUE TITLE
fine tune on validation set

### ISSUE THREAD

**Issue by sheshap** [princeton-vl/SimpleView#7]
Hi Team,

Can you please explain how is below fine-tuning is applicable in the case of ModelNet40? 
Aren't the validation and test set are same?

As discussed in the paper for the PointNet++ and SimpleView protocols, we need to first run an experiment to tune the number of epochs on the validation set. This could be done by first running the experiment <pointnet2/dgcnn>_<model_name>_valid_run_<seed>.yaml and then running the experiment <pointnet2/dgcnn>_<model_name>_run_<seed>.yaml. Based on the number of epochs achieving the best performance on the validation set, one could use the model trained on the complete training set to get the final test performance.

Thanks

**Comment by imankgoyal** [comment]
Hi, 

We create a separate validation set from the training set.

Thanks,
Ankit

**Comment by sheshap** [comment]
Thank you
